x-pack/plugins/aiops/common/error_utils.ts

```diff
@@ -0,0 +1,184 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { HttpFetchError } from '@kbn/core/public';
+import Boom from '@hapi/boom';
+import { isPopulatedObject } from '@kbn/ml-is-populated-object';
+
+export interface WrappedError {
+  body: {
+    attributes: {
+      body: EsErrorBody;
+    };
+    message: Boom.Boom;
+  };
+  statusCode: number;
+}
+
+export interface EsErrorRootCause {
+  type: string;
+  reason: string;
+  caused_by?: EsErrorRootCause;
+  script?: string;
+}
+
+export interface EsErrorBody {
+  error: {
+    root_cause?: EsErrorRootCause[];
+    caused_by?: EsErrorRootCause;
+    type: string;
+    reason: string;
+  };
+  status: number;
+}
+
+export interface DVResponseError {
```

Comment:
This file uses a `DV` prefix in several places, suggest to rename to `AiOps`.

Reply:
Updated in fd3cbb696ab1d3bc4f6c9cbcfd9f29124e986bf1